--- ggz-grubby-0.0.9/configure.ac.perl	2004-07-17 10:38:06.000000000 +0800
+++ ggz-grubby-0.0.9/configure.ac	2004-12-06 18:59:42.221317896 +0800
@@ -62,7 +62,7 @@
 	AC_GGZ_FIND_FILE("perl.h", $incs, perl_incdir)
 	perl_includes="$perl_incdir"
 
-	libs="/usr/local/lib /usr/lib"
+	libs="$inc /usr/local/lib /usr/lib"
 	AC_GGZ_FIND_FILE("libperl.so", $libs, perl_libdir)
 	perl_libraries="$perl_libdir"
 
